### DISCUSSION TITLE
Lenovo C330 Support?

### DISCUSSION THREAD

**Discussion by cdd100** [Maccraft123/Cadmium#145] [Q&A]
I have a Lenovo C330 and when I put Cadmium on a USB and boot from it, it goes black until I hit power + reload. Is support Coming?

**Comment by chinh4** [comment]
already added just build it yourself...

**Reply by cdd100** [reply]
I have the repo cloned on my other laptop, how should I edit the config to reflect my board?

**Reply by chinh4** [reply]
change rootfs to debian and it should be fine

**Reply by cdd100** [reply]
Thanks and sorry for any inconvenience this may have caused you!

**Reply by cdd100** [reply]
![16615659715921263538582289824688](https://user-images.githubusercontent.com/112119153/187010057-dcb5d545-135f-438e-b233-f7af72aee4b2.jpg)
Every time I try to compile the image, it gives me the error "8 out of 8 hunks ignored".

**Reply by cdd100** [reply]
Please help!

**Reply by chinh4** [reply]
try removing the conflicting patches
